packages/components/src/modal/README.md

```diff
@@ -1,30 +1,30 @@
-# Modal
+# Dialog
```

Comment:
Folder name should be updated as well. At the moment it's: ` packages/components/src/modal`.